head/firmware/main_proto.cpp

```diff
@@ -152,7 +152,7 @@ static tmc2130::configs::TMC2130DriverConfig motor_driver_configs_right{
                          .hstrt = 0x5,
                          .hend = 0x3,
                          .tbl = 0x2,
-                         .mres = 0x4},
+                         .mres = 0x3},
```

Comment:
We want to make this change on the proto too?

Reply:
there's no reason not to, really